api/v1alpha1/openapi.yaml

```diff
@@ -6889,13 +6889,15 @@ components:
       oneOf:
         - $ref: '#/components/schemas/OIDCProviderSpec'
         - $ref: '#/components/schemas/OAuth2ProviderSpec'
+        - $ref: '#/components/schemas/OpenShiftProviderSpec'
         - $ref: '#/components/schemas/AapProviderSpec'
```

Comment:
could `AAP` be on `OpenShift`? How would this map?

Reply:
i am actually not sure - we need to discuss each deployment configuration and understand what do we want to happen / what should be the default 

in that case would users receive a token from AAP or from openshift ?